src/Compilers/CSharp/Test/Semantic/FlowAnalysis/RegionAnalysisTests.cs

```diff
@@ -1593,7 +1593,7 @@ public static void Main()
             Assert.Null(GetSymbolNamesJoined(analysis.AlwaysAssigned));
             Assert.Equal("p", GetSymbolNamesJoined(analysis.Captured));
             Assert.Equal("i", GetSymbolNamesJoined(analysis.UnsafeAddressTaken));
-            Assert.Equal("", GetSymbolNamesJoined(analysis.VariablesDeclared));
+            Assert.Equal("<p>", GetSymbolNamesJoined(analysis.VariablesDeclared));
```

Comment:
This is a pre-existing behavior, the symbol is returned, assigning the name didn't change it. If you feel this is a bug, please open a separate issue to track it.